cis194/week1/zehua/Makefile

```diff
@@ -0,0 +1,2 @@
+run_test:
+	runhaskell -isrc -itest test/Spec.hs
```

Comment:
nit: I assume there is no stack set up. stack can be set up to run your tests by running `stack test` or with coverage `stack test --coverage`. It can also be set up to watch your file change and run test. And in development, using `ghcid` with `stack` maybe a good combination (https://www.parsonsmatt.org/2018/05/19/ghcid_for_the_win.html)

Reply:
added simple files for running with stack